UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/FssApiClient.cs

```diff
@@ -6,13 +6,10 @@ namespace UKHO.PeriodicOutputService.Common.Helpers
     public class FssApiClient : IFssApiClient
     {
         private HttpClient _httpClient;
-        private readonly IHttpClientFactory _httpClientFactory;
-
 
         public FssApiClient(IHttpClientFactory httpClientFactory)
         {
-            _httpClient = httpClientFactory.CreateClient();
-            _httpClientFactory = httpClientFactory;
+            _httpClient = httpClientFactory.CreateClient("DownloadClient");
```

Comment:
```suggestion
            _httpClient = httpClientFactory.CreateClient(FSSCLIENT);
```
and lets declare FssClient as constant variable FSSCLIENT

Reply:
Resolved. Created constant variable but value is kept as "DownloadClient" as it is used in POS and AIO.